### ISSUE TITLE
[Feature Request?] Defining Thresholds for gestures?

### ISSUE THREAD

**Issue by keiichiiownsu12** [Coffee2CodeNL/gebaar-libinput#28]
Has the ability to define a threshold, similar to fusuma and libinput-gestures for specific event, been implemented in gebaard yet, or due to come in a future release?

**Comment by LEXUGE** [comment]
Agree, without the ability of adjusting threshold of gestures, I found the actions were draggy instead of swift.

**Comment by osleg** [comment]
Basically the main reason why there is no threshold is that current implementation executes command on swipe end. 
This actually means that if you'd do short swift swipes they'd work *if* detected by your touchpad and libinput.

**Comment by osleg** [comment]
#31 should allow threshold for swipe gestures.

**Comment by osleg** [comment]
Implemented in my fork https://github.com/osleg/gebaar-libinput
